src/App/App.tsx

```diff
@@ -112,11 +115,14 @@ export const App: FunctionComponent = () => {
 
   const [showMetabaseHint, setShowMetabaseHint] = useState(true);
 
+  const anonymizedAccessDbIdLoadable = useAnonymizedAccessDbIdLoadable();
+  const anonymizedAccessDbId = useCachedLoadable(anonymizedAccessDbIdLoadable, 0);
```

Comment:
We can suspend here and not deal with the lazy loading. We need a `<Suspense>` wrapper somewhere high up the tree. Or maybe at tab level so we don't lose the whole UI.

Reply:
as mentioned in the Slack thread, the generic "+ New metabase tab" in current form is misleading, so it might actually be easier to revert it back to the default MB homepage (or remove)